app/models/otherExperience.py

```diff
@@ -0,0 +1,10 @@
+from app.models import *
+from app.models.term import Term
+
+class OtherExperience(baseModel):
+    activity = CharField()
+    term = ForeignKeyField(Term)
+    hours = IntegerField()
+    weeks = IntegerField()
+    service = CharField()

```

Comment:
'description' is probably a better name for this, and matches what you have in the UI